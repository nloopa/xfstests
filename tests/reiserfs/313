#! /bin/bash
# FS QA Test No. 313
#
# Check ctime and mtime are updated on truncate(2) and ftruncate(2)
#
# Regression test for commit:
# 3972f26 btrfs: update timestamps on truncate()
#
#-----------------------------------------------------------------------
# Copyright (c) 2013 Red Hat, Inc.  All Rights Reserved.
#
# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License as
# published by the Free Software Foundation.
#
# This program is distributed in the hope that it would be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program; if not, write the Free Software Foundation,
# Inc.,  51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
#-----------------------------------------------------------------------
#

seq=`basename $0`
seqres=$RESULT_DIR/$seq
echo "QA output created by $seq"

here=`pwd`
testfile=$TEST_DIR/testfile.$seq
status=1	# failure is the default!
trap "_cleanup; exit \$status" 0 1 2 3 15

_cleanup()
{
    cd /
    rm -f $testfile
}

# get standard environment, filters and checks
. ./common/rc
. ./common/filter

# real QA test starts here
_supported_fs generic
_supported_os IRIX Linux

echo "Silence is golden"

$here/src/t_truncate_cmtime $testfile 2>&1

status=0
exit
