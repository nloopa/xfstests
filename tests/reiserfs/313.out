QA output created by 313
Silence is golden
